src/layouts/components/Homepage/HeroBody.tsx

```diff
@@ -71,7 +71,7 @@ export const HeroBody = ({
   return (
     <>
       <Editable.Section spacing="1.25rem">
-        <FormControl>
+        <FormControl isRequired>
           <FormLabel>Notification Banner</FormLabel>
```

Comment:
I am not seeing the optional on InfoBar on staging, I think you mean Infopic in your description
![Screenshot 2023-08-29 at 5 38 46 PM](https://github.com/isomerpages/isomercms-frontend/assets/5507822/58e93b1c-8c86-4841-92b9-1426ddd0daec)

Reply:
updated PR desc